app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self,
```

Comment:
Place `self` on a new line so that there would be no arguments on the same line with an opening parenthesis as there are no arguments on the same line with a closing parenthesis.